src/components/orders-table/orders-data-table.component.tsx

```diff
@@ -47,15 +52,20 @@ const OrdersDataTable: React.FC<OrdersDataTableProps> = ({
   excludeColumns = [],
   fulfillerStatus,
   excludeCanceledAndDiscontinuedOrders = true,
+  useActivatedOnOrAfterDateFilter = true,
```

Comment:
@donaldkibet Is this the default behavior for all tabs or only the `active` tab?

Reply:
@pirupius that's correct. We are trying to avoid lifting more records that might not be relevant.